### DISCUSSION TITLE
@types/doubleclick-gpt dose not have Property 'enums'

### DISCUSSION THREAD

**Discussion by samizarraa** [DefinitelyTyped/DefinitelyTyped#56772] [Issues with a @types package]
Property 'enums' does not exist on type 'Googletag'
Angular

**Comment by samarmohan** [comment]
Can you provide more information about the issue you're having?

[a comment by an automated account is not shown]

**Comment by atwwei** [comment]
Just use [@types/googletag](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/googletag/index.d.ts)

**Comment by samizarraa** [comment]
Hi, Sorry I was sick
@types/doubleclick-gpt dose not have  Property 'enums'
